### PR TITLE
feat(item-layout): add component - vue3

### PR DESCRIPTION
**Pull request by juliodialpad**
# Vue 3 version of #1053

### CODE REVIEW

File: components/item_layout/item_layout_default.story.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <template #bottom>
+      <span v-html="bottom" />
+    </template>
+    <template #right>
+      <span v-html="right" />
+    </template>
+    <template #selected>
+      <span v-html="selected" />
+    </template>
+  </dt-item-layout>
```
Comment (braddialpad) [inline components/item_layout/item_layout_default.story.vue]: I think you forgot to update these to `$attrs` the component is not currently showing up.

File: components/item_layout/item_layout.stories.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  argTypes: argTypesData,
+  excludeStories: /.*Data$/,
+};
+
+// Templates
+const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtItemLayoutDefaultTemplate,
+);
```
Comment (braddialpad) [inline components/item_layout/item_layout.stories.js]: remove argtypes for vue 3

**Review by braddialpad — CHANGES_REQUESTED** [review]

File: components/item_layout/item_layout_default.story.vue (hunk 1 above)
Comment (juliodialpad) [reply components/item_layout/item_layout_default.story.vue]: > I think you forgot to update these to `$attrs` the component is not currently showing up.

Ah right, thought it was too easy 😅 good catch

**Comment by juliodialpad** [comment]
I'm merging this to update my dialtone PR before merging

**Pull request merged**